### PR TITLE
Revert "One-Way Windows"

### PR DESCRIPTION
**Pull request by clusterfack**
Reverts d3athrow/vgstation13#10429

### CODE REVIEW

**Comment by clusterfack** [comment]
I don't know the reason why, but we are having -insane- freezing once the round restarted

File: code/_onclick/hud/parallax.dm [hunk 1]
```diff
@@ -36,6 +36,12 @@ var/list/parallax_on_clients = list()
 
 /datum/hud/proc/update_parallax()
 	var/client/C = mymob.client
+	if(!parallax_initialized || C.updating_parallax) return
+
+	for(var/turf/T in range(get_turf(C.eye),C.view))
+		if(istype(T,/turf/space))
+			C.updating_parallax = 1
+			break
```
Comment (Intigracy) [inline code/_onclick/hud/parallax.dm:44]: What does any of this have to do with the windows?

Comment (clusterfack) [inline code/_onclick/hud/parallax.dm]: It's a reversion

Comment (Intigracy) [inline code/_onclick/hud/parallax.dm:44]: Yeah but reverting one commit should just be changing things changed in that commit.

Was this included in his changes on the PR?

Comment (clusterfack) [inline code/_onclick/hud/parallax.dm]: Yes, clearly

**Pull request merged**